src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Alert, Button, Checkbox, Col, ControlLabel, Form, FormGroup,
+         FormControl } from 'react-bootstrap'
+
+const LoginForm = ({ onSubmit, loginFailed }) => {
+  let emailInput, passwordInput, rememberInput
+  return (
+    <Form horizontal
+      onSubmit={(e) => {
+        e.preventDefault()
+        onSubmit(emailInput.value, passwordInput.value, rememberInput.checked)
+      }}
+    >
+      {
+        loginFailed ? (
+          <Alert bsStyle='danger'>
+            Invalid email or password.
+          </Alert>
+        ) : (
+          ''
+        )
+      }
+      <FormGroup controlId='formHorizontalEmail'>
+        <Col componentClass={ControlLabel} sm={2}>
+          Email
+        </Col>
+        <Col sm={10}>
+          <FormControl type='email' placeholder='Email'
+            inputRef={input => { emailInput = input }} />
```

Comment:
This is a bit ugly... isn't there any declarative way of binding the DOM object to a variable?